src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEventArgs.cs

```diff
@@ -1240,6 +1240,31 @@ internal AttMtuChangedEventArgs(string remoteAddress, int mtu)
         public int Mtu { get; }
     }
 
+    /// <summary>
+    /// An extended EventArgs class contains the service changed information.
```

Comment:
"contains the information about the service change" or "changed service"

Reply:
This class contains the information about the changed service.